app/scripts/acs/acs-graphs.js

```diff
@@ -18,8 +18,12 @@
         return module;
 
 
+        /**
+         * Replace text of length > max with an ellipsis
+         * Do no replacement if max === -1
+         */
         function addElipses(text, max) {
-            if (text.length > max) {
+            if (max !== -1 && text.length > max) {
```

Comment:
This was just a quickfix shortcut to disable the addition of the ellipsis, rather than setting max to a very high value. Theres a note in the comment above.

Sometimes I miss writing C. So I sneak it in when I can.